### PR TITLE
Bo branch

### PR DESCRIPTION
**Pull request by bo-pang-van**
two fixes: 
1. avoid reading "superseded" file in whisper. 
2. revise to epsilon in frame loss function.

### CODE REVIEW

File: src/training/custom_frame_level_loss.py [hunk 1]
```diff
@@ -51,7 +54,7 @@ def call(self, y_true: tf.Tensor, y_pred_frame: tf.Tensor) -> tf.Tensor:
         # Compute the loss
         loss = tf.cond(
             tf.equal(tf.size(y_true_masked), 0),
-            lambda: small_loss,
+            lambda: epsilon,
```
Comment (aanchan) [inline src/training/custom_frame_level_loss.py]: There isn't a significant change to the original cost function, we don't really need the case for 0. The cost function implementation is in the pdf document.

**Review by aanchan — CHANGES_REQUESTED** [review]
Hi @pp-never-cares can you update the cost function with the new version?
